posthog/temporal/tests/batch_exports/test_s3_batch_export_workflow.py

```diff
@@ -255,13 +311,15 @@ async def assert_clickhouse_records_in_s3(
 @pytest.mark.parametrize("compression", [None, "gzip", "brotli"], indirect=True)
 @pytest.mark.parametrize("exclude_events", [None, ["test-exclude"]], indirect=True)
 @pytest.mark.parametrize("batch_export_schema", TEST_S3_SCHEMAS)
+@pytest.mark.parametrize("file_format", ["JSONLines", "Parquet"])
```

Comment:
Should this use `FILE_FORMAT_EXTENSIONS.keys()` here and below, or is there some reason to avoid new formats being automatically included for testing? It seems like that would help catch (admittedly unlikely) errors of omission in `get_batch_export_writer`.

Reply:
New file formats could require some new handling in the `assert_cilckhouse_events_in_s3` test function, as we not only need to write them but also read them back when testing.

Reply:
Anyways, I think it would be good to fail and inform the person working on this they need to support testing.

Reply:
Eventually, file formats should be an enum. Kind of waiting for us to upgrade to Python 3.11 that introduces `StrEnum`, but I may do it anyways in a new PR once we start supporting new writers in other destinations.

Reply:
Resolved: using `FILE_FORMAT_EXTENSIONS.keys()` everywhere. We can later swap to an enum.